frontend/src/static/js/v_summary.js

```diff
@@ -278,6 +323,24 @@ window.vSummary = {
       this.getDates();
       this.sortFiltered();
     },
+    processFileFormats() {
+      const selectedColors = ['#ffe119', '#4363d8', '#3cb44b', '#f58231', '#911eb4', '#46f0f0', '#f032e6',
+        '#bcf60c', '#fabebe', '#008080', '#e6beff', '#9a6324', '#fffac8', '#800000', '#aaffc3', '#808000', '#ffd8b1',
```

Comment:
double indent

Reply:
@eugenepeh adding this double indentation seems to be causing CI to fail, i think this eslint rule is not enforced currently?

Reply:
@apoorva17 yes most likely the case. Can I trouble you to add in the rule for this as well?
Same as before can be a simple PR, no elaborated message is needed.

Reply:
Sure, I've created the PR.